cluster/src/main/java/com/hartwig/pipeline/cleanup/Cleanup.java

```diff
@@ -49,14 +57,40 @@ public void run(SomaticRunMetadata metadata) {
                         .getJobId()
                         .startsWith(tumorRun.id())) {
                     LOGGER.debug("Deleting complete job [{}]", job.getReference().getJobId());
-                    jobs.delete(arguments.project(), arguments.region(), job.getReference().getJobId()).execute();
+                    deleteJob(jobs, job);
+
+                    Job existingJob = Failsafe.with(new RetryPolicy<>().handleResultIf(Objects::nonNull)
+                            .onFailedAttempt(objectExecutionCompletedEvent -> deleteJob(jobs, job))
+                            .withDelay(Duration.ofSeconds(3))
```

Comment:
Strikes me as a bit odd not to have a constant for the delay if the max retries gets one.